### ISSUE TITLE
 Makefile does not support out-of-tree builds 

### ISSUE THREAD

**Issue by josemonsalve2** [OpenMP-Validation-and-Verification/OpenMP_VV#5]
From @Meinersbur 
```bash
$ make -f ../sollve_vv-src/Makefile
../sollve_vv-src/Makefile:14: sys/make/make.def: No such file or directory
make: *** No rule to make target `sys/make/make.def'.  Stop.
```

**Comment by josemonsalve2** [comment]
https://bitbucket.org/crpl_cisc/sollve_vv/issues/1/makefile-does-not-support-out-of-tree

**Comment by josemonsalve2** [comment]
@Meinersbur I have added this issue here from Bitbucket. You created it in the first place

**Comment by andrewkallai** [comment]
`make --directory=path/to/OpenMP_VV`
Using make with this flag will instruct it to use the makefile in that location.
